src/components/sections/About.section.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import AboutAirBalloon from '../../assets/about/AboutAirBalloon.svg';
+import AboutSection from '../../assets/about/AboutMeSection.svg';
+
+const About: React.FC = () => {
+    const [offsetY, setOffsetY] = useState(0);
+
+    const handleScroll = () => {
+        setOffsetY(window.scrollY * 0.45); // Handles the speed at which the balloon moves
+    };
+
+    useEffect(() => {
+        window.addEventListener('scroll', handleScroll);
+
+        return () => window.removeEventListener('scroll', handleScroll);
+    }, []);
+    return (
+        <div className="relative bg-gradient-to-b from-peachPuff to-brightUbe pb-10 pt-48">
+            {/* MAIN SVG (Background) */}
+            <div className="relative left-[-4%] w-[115%] overflow-hidden">
```

Comment:
if you import the svg well, then you don't need to manually change the size to fit the screen though.
That's how I would do it 👍 
Otherwise, magic number would appear again hhh

Reply:
You gotta show me how you fix the sizing for the SVGs. I did it, but definitely not efficiently 😅 It was a bit of a pain but it should be fixed now!

Reply:
This is how I did it hhh

1. Put the exported background image to the desired view.
![Screenshot 2024-02-26 at 8 15 17 PM](https://github.com/LaurierHawkHacks/Landing/assets/98545971/ed310df5-f89b-4089-937d-83abc3276499)

2. Turn the background components into their own "group"
![Screenshot 2024-02-26 at 8 17 38 PM](https://github.com/LaurierHawkHacks/Landing/assets/98545971/dc34f3ca-2d88-45a7-8b1d-080963c6fdee)

3. Export the background, with the following setting...
![Screenshot 2024-02-26 at 8 20 51 PM](https://github.com/LaurierHawkHacks/Landing/assets/98545971/7e34ca00-91a9-41c4-8477-c0877468db63)